code/game/turfs/simulated/walls_mineral.dm

```diff
@@ -80,8 +80,8 @@
 	name = "plasma wall"
 	desc = "A wall with plasma plating. This is definately a bad idea."
 	icon_state = "plasma0"
-	walltype = "plasma"
-	mineral = "plasma"
+	walltype = PLASMA
+	mineral = PLASMA

```

Comment:
inappropriate replacement (mineral ID and walltype, both need to be refactored separately)